src/components/features/ProductSearch/ProductSearch.module.scss

```diff
@@ -58,3 +58,14 @@
     }
   }
 }
+
+
+@media (max-width: 575px) {
```

Comment:
Zrób merge do `master` w tedy pojawi Ci się w global.scss `mixin`, używajmy mixinów, tylko trzeba go będzie delikatnie poprawić bo ten co jest na masterze jest nazwany `mobile` a tak naprwdę tyczy się tabletów, to zmień mu nazwę i zrób swojego `mobile` ;)

Reply:
Wprowadziłam zmiany @kamilwilk95 :)